suit/templatetags/suit_tags.py

```diff
@@ -112,3 +112,12 @@ def add_preserved_filters(*args, **kwargs):
     @register.filter
     def admin_urlquote(value):
         return quote(value)
+
+if django_version >= (5, 1):
+    @register.filter
+    def length_is(value, arg):
```

Comment:
could we use filter `length` instead? as mentioned here https://docs.djangoproject.com/en/dev/releases/4.2/#id1